src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/CassandraEventsSink.scala

```diff
@@ -65,10 +68,26 @@ object CassandraEventsSink{
           aggregators.foreach(aggregator => {
             val eventName = aggregator.FortisTargetTablename
 
-            Timer.time(Telemetry.logSinkPhase(s"aggregate_$eventName", _, _, batchSize)) {
+            Timer.time((completed, time)=> println(s"Ran aggregator ${aggregator.getClass.getSimpleName} in ${time} nanoseconds.")) {
```

Comment:
Since you are doing `aggregator.getClass` in a `foreachRDD` closure, won't the aggregator class need to be serialized?  (https://forums.databricks.com/questions/382/why-is-my-spark-streaming-application-throwing-a-n.html) 
Maybe there is a different way to provide the name of the aggregator without needing the class to be serialized.